### PR TITLE
Allow no support package in LocalPackageMeshLoader

### PR DESCRIPTION
**Pull request by yck011522**
This will allow the LocalPackageMeshLoader to process robot 
 packages that does not have a support_package sub folder. This is used for loading packages from the local_cache_directory of the RosFileServerLoader. 

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [x] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by yck011522 — COMMENTED** [review]
For the interest of not changing signature, I did not change the init signature to have `support_package = None`

File: src/compas_robots/resources/basic.py [hunk 1]
```diff
@@ -165,7 +165,10 @@ def __init__(self, path, support_package):
         super(LocalPackageMeshLoader, self).__init__()
         self.path = path
         self.support_package = support_package
-        self.schema_prefix = "package://" + self.support_package + "/"
+        if support_package is None or support_package == "":
```
Comment (gonzalocasas) [inline src/compas_robots/resources/basic.py:168]: The pythonic way is to rely on truthy testing:
```suggestion
        if support_package:
```

File: src/compas_robots/resources/basic.py [hunk 2]
```diff
@@ -165,7 +165,10 @@ def __init__(self, path, support_package):
...
+            self.schema_prefix = "package://"
```
Comment (gonzalocasas) [inline src/compas_robots/resources/basic.py:169]: Please add a note in the docstring indicating `support_package` is optional, and add a default `None` to the param itself.

**Review by gonzalocasas — COMMENTED** [review]
Please add a test to verify that this works, because I see there are calls to `build_path` that rely on `support_package` being there and while they will still work now, it feels a bit fragile.

**Review by yck011522 — COMMENTED** [review]
> Please add a test to verify that this works, because I see there are calls to `build_path` that rely on `support_package` being there and while they will still work now, it feels a bit fragile. 

I can see that there are calls to `build_path` , maybe I should make a switch there too to not include `support_package` if it is empty.

Regarding test, I'm not sure what you are refering to as fragile, can you elaborate more or explain how you would design the test? @gonzalocasas

**Pull request merged**